kombu/asynchronous/semaphore.py

```diff
@@ -28,13 +37,18 @@ class LaxBoundedSemaphore:
         HELLO 3
     """
 
-    def __init__(self, value):
+    def __init__(self, value: int) -> None:
         self.initial_value = self.value = value
-        self._waiting = deque()
+        self._waiting: deque = deque()
```

Comment:
`Deque[<Type>]`?

Reply:
Yeah you're right. Think the correct type can use the ParamSpec actually:

```python
Deque[Tuple[P, P.args, P.kwargs]]
```